index.html

```diff
@@ -46,6 +46,10 @@ <h1 class="title-lg pd-point8-lt nav-sub-head">
 			<button class="btn secondary-text-btn-lg-nav">
 				Alert</button>
 		</a>
+		<a href="#buttons">
+			<button class="btn secondary-text-btn-lg-nav">
```

Comment:
try to use either anchro tag or button. Dont use both. If you want to make your anchor look like button you can use 'btn' classed on you <a/> tag

Reply:
Thanks for Adding will do it !